### PR TITLE
docs: more in lag/at

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: hybridse/src/udf/default_defs/window_functions_def.cc [hunk 1]
```diff
@@ -96,8 +96,9 @@ void RegisterBaseListLag(UdfLibrary* lib) {
     lib->RegisterExternal("lag")
         .doc(R"(
             @brief Returns value evaluated at the row that is offset rows before the current row within the partition. Offset is evaluated with respect to the current row
+            The offset in window is `nth_value()`, not `lag()/at()`.
             Note: This function equals the `at()` function.
-            But old `at()` is start from the last row of window(may not be the current row)
+            But old `at()` is start from the last row of window(may not be the current row), it's more like `nth_value()`
```
Comment (aceforeverd) [inline hybridse/src/udf/default_defs/window_functions_def.cc]: old `at` confuses current `at`,  note it as `at version < 0.5.0 `

Comment (aceforeverd) [reply hybridse/src/udf/default_defs/window_functions_def.cc]: or no need to explain this

Comment (vagetablechicken) [reply hybridse/src/udf/default_defs/window_functions_def.cc]: ok

**Review by aceforeverd — APPROVED** [review]

File: hybridse/src/udf/default_defs/window_functions_def.cc [hunk 2]
```diff
@@ -96,8 +96,9 @@ void RegisterBaseListLag(UdfLibrary* lib) {
     lib->RegisterExternal("lag")
         .doc(R"(
             @brief Returns value evaluated at the row that is offset rows before the current row within the partition. Offset is evaluated with respect to the current row
+            The offset in window is `nth_value()`, not `lag()/at()`.
             Note: This function equals the `at()` function.
```
Comment (aceforeverd) [inline hybridse/src/udf/default_defs/window_functions_def.cc:101]: add a newline before `Note`, this is a markdown string

**Review by aceforeverd — CHANGES_REQUESTED** [review]

**Review by aceforeverd — APPROVED** [review]

**Review by dl239 — APPROVED** [review]

**Pull request merged**